plugins/out_stackdriver/stackdriver.c

```diff
@@ -1104,6 +1116,7 @@ static int pack_json_payload(int insert_id_extracted,
         monitored_resource_key,
         local_resource_id_key,
         ctx->labels_key,
+        ctx->trace_key,
```

Comment:
We should also add `ctx->severity_key` here, but this will be a breaking change if someone relies on it in `jsonPayload`. If we follow the semantic versioning, this will require moving changing the major version. I'll leave it to your judgement.